src/DataProvider/ItemDataProviderInterface.php

```diff
@@ -26,11 +26,12 @@
      * @param string      $resourceClass
      * @param string|null $operationName
      * @param int|string  $id
-     * @param bool        $fetchData
+     * @param array       $context       Context can contain serialization context
```

Comment:
The comment about serialization groups should go into EagerLoadingExtension.

The comment about `fetch_data` should go into the implementation of the data provider.